### PR TITLE
add example for clarity

### PR DESCRIPTION
**Pull request by guillermooo**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[aspnetcore/fundamentals/routing.md](https://github.com/guillermooo/Docs/blob/patch-5/aspnetcore/fundamentals/routing.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C12%5C6%5C6a0d29ae-bb37-09f1-c55a-4cfa862c6784%5CPullRequest%5C201612061037264299-2326%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: aspnetcore/fundamentals/routing.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Routing uses *routes* (implementations of `IRouter`) to:
 
 * generate URLs used in responses
 
-Generally an app has a single collection of routes. The route collection is processed in order. Requests look for a match in the route collection by . Responses use routing to generate URLs.
+Generally an app has a single collection of routes. The route collection is processed in order. Requests look for a match in the route collection by . Responses use routing to generate URLs (for example, for redirection).
```
Comment (guillermooo) [inline aspnetcore/fundamentals/routing.md]: That's a pre-existing bug. I couldn't figure it out either, but I guess something along the lines of "by runing the XXXAsync method"? (I can't remember the method name right now.)

**Comment by Rick-Anderson** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[aspnetcore/fundamentals/routing.md](https://github.com/guillermooo/Docs/blob/patch-5/aspnetcore/fundamentals/routing.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C12%5C6%5C6a0d29ae-bb37-09f1-c55a-4cfa862c6784%5CPullRequest%5C201612061443062092-2326%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: aspnetcore/fundamentals/routing.md (hunk 1 above)
Comment (guillermooo) [inline aspnetcore/fundamentals/routing.md]: Ooops. Missed this comment. I've edited a bit more in the meantime. Let me know what you think. I can roll back the changes and fix as suggested instead.

**Comment by Rick-Anderson** [comment]
@rynowak  please review this paragraph change

**Comment by Rick-Anderson** [comment]
@guillermooo  Thanks!

**Comment by rynowak** [comment]
Looks fine

**Pull request merged**